frontend/src/components/basicInfo/BasicProjectInfo.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+
+import { Card } from '@/lib/primevue';
+
+// Props
+const { assignee, createdBy, activityId } = defineProps<{
+  assignee?: string;
```

Comment:
Doesn't need to be optional, it will always receive a string from the parent. Sometimes a name sometime "To be assigned"